src/__tests__/test-utils-selectors.test.tsx

```diff
@@ -0,0 +1,141 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import fs from 'fs';
+import Path from 'path';
+import glob from 'glob';
+import { transformSync, types, PluginObj, NodePath } from '@babel/core';
```

Comment:
This module is not declared in our dependencies. You should `npm install --save-dev @babel/core` to not break our builds.